python/cudf/cudf/tests/test_datetime.py

```diff
@@ -2148,3 +2148,11 @@ def test_daterange_pandas_compatibility():
             "2010-01-01", "2010-02-01", periods=10, name="times"
         )
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize("code", ["z", "Z"])
+def test_format_timezone_not_implemented(code):
+    with pytest.raises(NotImplementedError):
+        cudf.to_datetime(
+            ["2020-01-01 00:00:00 UTC"], format=f"%Y-%m-%d %H-%M-%S %{code}"
```

Comment:
```suggestion
            ["2020-01-01 00:00:00 UTC"], format=f"%Y-%m-%d %H:%M:%S %{code}"
```

Format string doesn't match the provided date string without it (and once timezones are implemented, we would expect this test to `raise ValueError` because of it).

Reply:
Thanks for the catch